modules/jdbc-test/src/test/java/org/testcontainers/jdbc/JDBCDriverTest.java

```diff
@@ -84,6 +88,27 @@ public void test() throws SQLException {
         }
     }
 
+    @Test
+    public void shouldStopContainerWhenAllConnectionsClosed() throws SQLException {
```

Comment:
please move these two tests to a separate Test class, otherwise, they will execute for *every* parameterized data entry